src/common/file_linux.c

```diff
@@ -222,6 +225,18 @@ device_dax_alignment(const char *path)
 		goto out;
 	}
 
+	/* just in case someone is using kernel < 4.9 */
+	if (size != 4096 && size != 2 * MiB && size != GiB) {
```

Comment:
Alternatively one might write something like:
`if ((size & (size - 1)) != 0)`
and than no need to define MiB and GiB